src/App.js

```diff
@@ -0,0 +1,144 @@
+import './App.css';
+import React, { useState }  from 'react';
+import  women from './img/illustration-woman-online-desktop.svg';
+import  bg from './img/bg-pattern-mobile.svg';
+import  bg1 from './img/bg-pattern-desktop.svg';
+import box from './img/illustration-box-desktop.svg';
+import arrowIcon from './img/icon-arrow-down.svg';
+
+const App = () => {
+  return (
+    <MainDiv />
+  );
+}
+
+
+const MainDiv = () => {
+  return (
+    <div className='main'>
+      <LeftDiv />
+      <img className='svgThird' src={box} alt='box'></img>
+      <RightDiv />
+    </div>
+  );
+}
+
+
+const LeftDiv = () => {
+  return (
+    <div className='main__pic'>
+     <img className='svgFirst' src={bg1} alt='bg'></img>
+      <img className='svgSecond' src={women} alt='women'></img>
+      <img className='svgForth' src={bg} alt='box'></img>
+    </div>
+  );
+}
+
+
+const RightDiv = () => {
+  return (
+    <div className='m-div'>
+      <div className='FAQ'>
+      <h1>FAQ</h1>
+    </div>
+      <Accardion />
+    </div>
+
+  );
+}
+
+
+
+
+
+const Accardion = () => {
+  const questions = {
+    first: {
+      question: 'How many team members can I invite?',
+      answer: 'You can invite up to 2 additional users on the Free plan. There is no limit on team members for the Premium plan.'
+    },
+    second: {
+      question: 'What is the maximum file upload size?',
+      answer: 'No more than 2GB. All files in your account must fit your allotted storage space.'
+    },
+    third: {
+      question: 'How do I reset my password?',
+      answer: 'Click “Forgot password” from the login page or “Change password” from your profile page. A reset link will be emailed to you.'
+    },
+    forth: {
+      question: 'Can I cancel my subscription?',
+      answer: 'Yes! Send us a message and we’ll process your request no questions asked.'
+    },
+
+    fifth: {
+      question: ' Do you provide additional support? ',
+      answer: 'Chat and email support is available 24/7. Phone lines are open during normal business hours.'
+    }
+  }
+
+  const listItems = Object.keys(questions).map(
+    (key) => {
+      return (
+        <AccordionItem id={key} obj={questions[key]} />
+      )
+    }
```

Comment:
1. Return is not needed here if u use parentheses instead of curly braces
2.`key` prop is mandatory https://reactjs.org/docs/lists-and-keys.html#keys
3. `AccordionItem` should receive `data|item|question` prop instead of `obj`
```
(key) => (
	<AccordionItem key={key} id={key} data={questions[key]} />
)
```